pkg/controller/elasticsearch/filesettings/file_settings.go

```diff
@@ -145,7 +145,11 @@ func mutateSnapshotRepositorySettings(snapshotRepository map[string]interface{},
 	}
 	switch snapshotRepository["type"] {
 	case "azure", "gcs", "s3":
-		settings["base_path"] = fmt.Sprintf("snapshots/%s", suffix)
+		basePath, ok := settings["base_path"].(string)
```

Comment:
Should we also update the documentation to explain how `base_path` is updated by the operator?

Reply:
Yes.